index.php

```diff
@@ -23,46 +23,55 @@
         </div>
     </aside>
 
-    <main class="l-side-by-side__item" role="main">
+    <div class="l-side-by-side__item">
 
-        <h2 class="heading heading--main h-spacing-base">Pets</h2>
+        <main class="l-main" role="main">
 
-        <ul class="l-grid">
+            <h2 class="heading heading--main h-spacing-base">Pets</h2>
 
-        <?php
-            // get the posts
-            $posts = get_posts();
-                if($posts){
-                    foreach ($posts as $key => $post){
-        ?>
-            <li class="l-grid__item l-grid__item--4-col">
+            <ul class="l-grid">
 
-                <div class="c-card">
+            <?php
+                // get the posts
+                $posts = get_posts();
+                    if($posts){
+                        foreach ($posts as $key => $post){
+            ?>
+                <li class="l-grid__item l-grid__item--4-col">
 
-                    <div class="c-card__header">
-                        <h3 class="heading heading--main"><a href="/pet-details.php?pet=<?php echo substr($key, 1); ?>">Pet Name</a></h3>
-                    </div>
+                    <a class="c-card" href="/pet-details.php?pet=<?php echo substr($key, 1); ?>">
 
-                    <div class="c-card__body">
-                        <ul>
-                            <li>Species: <?php echo $post->breed; ?></li>
-                            <li>Colour: <?php echo $post->colour; ?></li>
-                            <li>Other info</li>
-                        </ul>
-                        <a href="/">I've found this pet!</a>
-                    </div>
-                </div>
-                <!-- .c-card -->
+                        <div class="c-card__header">
+                            <h3 class="heading heading--main">Pet Name <?php echo $post->name; ?></h3>
+                        </div>
 
-            </li>
+                        <div class="c-card__body">
+                            <ul>
+                                <li>Species: <?php echo $post->species; ?></li>
+                                <li>Breed: <?php echo $post->breed; ?></li>
+                                <li>Size: <?php echo $post->size; ?></li>
+                                <li>Colour: <?php echo $post->colour; ?></li>
+                                <li>Collar: <?php echo $post->collar; ?></li>
+                                <li>Location: <?php echo $post->location; ?></li>
+                            </ul>
+                        </div>
+                    </a>
+                    <!-- .c-card -->
```

Comment:
I would wrap these in if statements, unless we're going to force everything to be required in the DB - I did this same template before and got loads of errors for things like 'name' (currently not in the form)